test/ui-test/testSuites/suite_settings/tst_userIdentity/test.feature

```diff
@@ -26,8 +27,8 @@ Feature: User Identity
         And the user's bio should be "Hello, I am super tester!"
         And the user's social links should be: "twitter_handle", personal site: "status.im", "customLink": "customUrl"
 
-	@merge
-    Scenario: The user sets can change own display name in profile popup
+	@mayfail
```

Comment:
Can you add a comment here as well to explain why it may fail?

Reply:
Same comment than before. Added comment in code.